src/Compilers/Core/Portable/PEWriter/FullMetadataWriter.cs

```diff
@@ -387,10 +391,11 @@ protected override void CreateIndicesForNonTypeMembers(ITypeDefinition typeDef)
 
             foreach (MethodImplementation methodImplementation in typeDef.GetExplicitImplementationOverrides(Context))
             {
+                // PROTOTYPE(refout) Do something here?
```

Comment:
>// PROTOTYPE(refout) Do something here? [](start = 16, length = 39)

I think we should keep explicit method implementations so that compilers can properly figure out if interface members/abstract members are implemented.